WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -164,6 +171,18 @@ final class OrderDetailsDataSource: NSObject {
     func configureResultsControllers(onReload: @escaping () -> Void) {
         resultsControllers.configureResultsControllers(onReload: onReload)
     }
+
+    func shippingLabelOrderItem(at indexPath: IndexPath) -> AggregateOrderItem? {
+        guard let shippingLabel = shippingLabel(at: indexPath) else {
+            return nil
+        }
+
+        guard let orderItem = shippingLabelOrderItemsAggregator?.orderItems(of: shippingLabel)[safe: indexPath.row] else {
+            return nil
+        }
+
+        return orderItem
```

Comment:
Kudos for using `safe`! 🎉 

ℹ️ It looks like we can shorten this. 🙂 

```suggestion
        return shippingLabelOrderItemsAggregator?.orderItems(of: shippingLabel)[safe: indexPath.row]
```

Reply:
It's nice optimizing the LOC in an already big class 😄 updated in https://github.com/woocommerce/woocommerce-ios/pull/3197/commits/7dbe1f3ebf33f3e571285748bc76007225b1618f